### PR TITLE
Add note about route configuration

### PR DESCRIPTION
**Pull request by benadam11**

### CODE REVIEW

File: content/sites/start-from-scratch.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ To start from scratch to create a Workers Site, follow these steps:
      zone_id = "42ef.."
      route = "example.com/*"
      ```
+   (Note: the wildcard selector is important to ensure that paths to your assets match and are loaded properly)
```
Comment (signalnerve) [inline content/sites/start-from-scratch.md]:
```suggestion
   (Note: the wildcard selector is important to ensure that paths to your assets match, and are loaded properly) 
```

Comment (signalnerve) [reply content/sites/start-from-scratch.md]: Just a quick little suggestion here, once that's added, happy to merge! Thank you :)

Comment (benadam11) [reply content/sites/start-from-scratch.md]: Awesome!

File: content/sites/start-from-scratch.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ To start from scratch to create a Workers Site, follow these steps:
      zone_id = "42ef.."
      route = "example.com/*"
      ```
+   (Note: the wildcard selector is important to ensure that paths to your assets match, and are loaded properly)
```
Comment (victoriabernard92) [inline content/sites/start-from-scratch.md]:
```suggestion
   (Note: Check out documentation on [Routes](/workers/about/routes) to configure `route` properly) 
```

**Comment by benadam11** [comment]
@victoriabernard92 - there might be a better place to add clarity than here, but it was really missing the wildcard when trying to set up a Workers Site caused me a lot of confusion and wasn't really discussed anywhere in the Worker Sites documentation. Happy to make any changes that would add clarity :)

File: content/sites/start-from-scratch.md (hunk 2 above)
Comment (benadam11) [reply content/sites/start-from-scratch.md]: That is definitely better

**Comment by victoriabernard92** [comment]
I see I was a little out of context. Thanks for this @benadam11 !

**Pull request merged**